scripts/snap

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+echo "FAKE SNAP: Trying to execute 'snap $*' but snap are not supported on Docker, skipping it sorry."
```

Comment:
Can this just be faked like this? won't we miss some packages that way?
I found this thread https://forum.snapcraft.io/t/snapd-in-docker/177/16 - don't know if that's helpful.

Reply:
I know, it.’a a bitter hack, but there isn’t any other way to fix it, Snaps cannot be installed on a docker build context cause it needs a system daemon running and it is another container management under the hood.
I will open a PR to the virtual-environments project to remove them,  but luckily it is just related to two installers:  https://github.com/actions/virtual-environments/search?q=snap just git and kubernetes-tools.

I guess here we can decide to just fake the snap binary like this or the more conservative way to leave it fails.
I don’t have a strong opinion on this.

Reply:
The first PR to remove the snaps: https://github.com/actions/virtual-environments/pull/1513/files

Reply:
The second one to remove kustomize: https://github.com/actions/virtual-environments/pull/1515

Anyway i think that here we are safe to go, did you have a chance to test it ?

Reply:
I think that's all I want changed, and then this is good to go.

Reply:
Done, fake snap binary removed, until we'll have the PRs merged, the installers that uses snaps will fail, for now just git and kubernetes-tools.